cmd/otel-agent/command/command.go

```diff
@@ -63,12 +64,18 @@ func makeCommands(globalParams *subcommands.GlobalParams) *cobra.Command {
 }
 
 const configFlag = "config"
+const coreConfigFlag = "core-config"
+const syncDelayFlag = "sync-delay"
```

Comment:
Note: these must be flags because they cannot be sourced from any config - this is required pre-config init (environment variables could be an alternative too).

Reply:
Lets move this to env var for v1. This is changing api for collector will make it hard for us to migrate to ocb